src/botPage/bot/Interpreter.js

```diff
@@ -149,8 +149,10 @@ export default class Interpreter {
     }
     terminateSession() {
         this.$scope.api.disconnect();
-        globalObserver.emit('bot.stop');
         this.stopped = true;
+
+        globalObserver.emit('bot.stop');
```

Comment:
@aaron-binary why can we just change the buttons state when we emit bot.stop, why we need to keep it in a state?